src/deepqmc/train.py

```diff
@@ -127,11 +132,20 @@ def train(  # noqa: C901
     if workdir:
         workdir = f'{workdir}/{mode}'
         chkpts = CheckpointStore(workdir, **(chkpts_kwargs or {}))
-        writer = tensorboard.summary.Writer(workdir)
+        global_writer = tensorboard.summary.Writer(f'{workdir}')
+        writers = [
+            tensorboard.summary.Writer(f'{workdir}/{i}') for i in range(len(sampler))
+        ]
         log.debug('Setting up HDF5 file...')
         h5file = h5py.File(f'{workdir}/result.h5', 'a', libver='v110')
         h5file.swmr_mode = True
-        table = H5LogTable(h5file)
+        tables = []
+        for i, R in enumerate(sampler.nuclear_configurations()):
+            group = h5file.require_group(str(i))
+            group.attrs.create('geometry', R.tolist())
+            table = H5LogTable(group)
+            table.resize(init_step)
+            tables.append(table)
```

Comment:
Same here, we probably want to produce only outputs aggregated over the dataset eventually, but we can discuss this at a later date

Reply:
If we are thinking about aggregated values to track rather than per-configuration values, the var(E_loc) could be really useful as it has to go to 0 globally